projects/cdk/directives/autofilled/autofilled.directive.ts

```diff
@@ -1,53 +1,24 @@
-import {Directive, ElementRef, Inject, Output} from '@angular/core';
-import {typedFromEvent} from '@taiga-ui/cdk/observables';
-import {TuiDirectiveStylesService} from '@taiga-ui/cdk/services';
-import {map, skip} from 'rxjs/operators';
+import {Directive, EventEmitter, HostBinding, HostListener, Output} from '@angular/core';
 
-const ON = 'tuiAutofillOn';
-const OFF = 'tuiAutofillOff';
-const STYLE = `
-@keyframes ${ON} {
-    from {
-        content: '1';
-    }
-
-    to {
-        content: '2';
-    }
-}
-
-@keyframes ${OFF} {
-    from {
-        content: '2';
-    }
-
-    to {
-        content: '1';
-    }
-}`;
-
-// @dynamic
 @Directive({
     selector: '[tuiAutofilledChange]',
-    host: {
-        class: 'tui-autofill',
-    },
+    host: {class: 'tui-autofill'},
```

Comment:
I believe we no longer need this class, do we?

Reply:
Yes, `transition: box-shadow 0.01s;` we need to define rules only for those elements who `input.tui-autofill` or `.tui-autofill input`